### PR TITLE
Implementation of the classes ConvertPPTtoImage and CriteriaPPTtoImage

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**
OK Guys, I implemented ConvertPPTtoImage and CriteriaPPTtoImage classes.

### CODE REVIEW

File: src/main/java/com/fundation/webservice/model/ConvertPdfToImage.java [hunk 1]
```diff
@@ -24,9 +24,12 @@
  */
 public class ConvertPdfToImage implements IConvert {
     CriteriaPdfToImage criterion;
-
     ConvertPdfToImage(CriteriaPdfToImage criterion) {
         this.criterion = criterion;
+
+    ConvertPdfToImage(CriteriaPdfToImage criterio){
+            this.criterion = criterio;
+
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/ConvertPdfToImage.java:32]: Is this space necessary?

File: src/main/java/com/fundation/webservice/model/ConvertPPTtoImage.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        try {
+            File file = new File(criterion.getSrcPath());
+            XMLSlideShow ppt = new XMLSlideShow(new FileInputStream(file));
+            Dimension pgsize = ppt.getPageSize();
+            List<XSLFSlide> slide = ppt.getSlides();
+            BufferedImage img = null;
+            for (int i = 0; i < slide.size(); i++) {
+                img = new BufferedImage(pgsize.width, pgsize.height, BufferedImage.TYPE_INT_RGB);
+                Graphics2D graphics = img.createGraphics();
+
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/ConvertPPTtoImage.java:48]: space is necessary?

File: src/main/java/com/fundation/webservice/model/ConvertPdfToImage.java [hunk 3]
```diff
@@ -24,9 +24,12 @@
  */
 public class ConvertPdfToImage implements IConvert {
     CriteriaPdfToImage criterion;
-
     ConvertPdfToImage(CriteriaPdfToImage criterion) {
         this.criterion = criterion;
+
+    ConvertPdfToImage(CriteriaPdfToImage criterio){
+            this.criterion = criterio;
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/ConvertPdfToImage.java:31]: fix variable name 'criterio' please

File: src/main/java/com/fundation/webservice/model/ConvertPPTtoImage.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                graphics.fill(new Rectangle2D.Float(0, 0, pgsize.width, pgsize.height));
+
+                //render
+                slide.get(i).draw(graphics);
+                FileOutputStream out = new FileOutputStream(criterion.getDestPath() + i + "." + criterion.getExt());
+                javax.imageio.ImageIO.write(img, criterion.getExt(), out);
+                ppt.write(out);
+                out.close();
+            }
+
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/ConvertPPTtoImage.java:59]: space is necessary?

File: src/main/java/com/fundation/webservice/model/ConvertPPTtoImage.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/ConvertPPTtoImage.java:64]: space is necessary?

File: src/main/java/com/fundation/webservice/model/ConvertPPTtoImage.java (hunk 2 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/com/fundation/webservice/model/ConvertPPTtoImage.java:48]: yes

File: src/main/java/com/fundation/webservice/model/ConvertPPTtoImage.java [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+            Dimension pgsize = ppt.getPageSize();
+            List<XSLFSlide> slide = ppt.getSlides();
+            BufferedImage img = null;
+            for (int i = 0; i < slide.size(); i++) {
+                img = new BufferedImage(pgsize.width, pgsize.height, BufferedImage.TYPE_INT_RGB);
+                Graphics2D graphics = img.createGraphics();
+
+                graphics.setPaint(Color.white);
+                graphics.fill(new Rectangle2D.Float(0, 0, pgsize.width, pgsize.height));
+
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/ConvertPPTtoImage.java:51]: space is necessary?

File: src/main/java/com/fundation/webservice/model/ConvertPPTtoImage.java (hunk 5 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/com/fundation/webservice/model/ConvertPPTtoImage.java:64]: Ok, I fix right away

**Pull request merged**